homework/index.js

```diff
@@ -33,10 +33,81 @@
   function main(url) {
     fetchJSON(url, (err, data) => {
       const root = document.getElementById('root');
+
       if (err) {
         createAndAppend('div', root, { text: err.message, class: 'alert-error' });
       } else {
-        createAndAppend('pre', root, { text: JSON.stringify(data, null, 2) });
+        const header = createAndAppend('header', root, { id: 'header' });
+        createAndAppend('p', header, { text: 'HYF Repositories' });
+        const select = createAndAppend('select', header, { id: 'select' });
+
+        // CREATE EVENT LISTENER
+        select.addEventListener(
+          'change',
+          () => {
+            const selected = select.selectedIndex;
+            const repositoryValue = data[selected].name;
+            const repoLink = data[selected].html_url;
+            if (repositoryValue === null) {
+              document.getElementById('property_0').innerText = '-';
```

Comment:
Instead of calling it `property_0`, why not call it `property_repository`? That's much easier to read.

Reply:
IDs are changed with more semantic ones.